### PR TITLE
Add addendum results to Lifestyle Section

### PR DESCRIPTION
**Pull request by nsabahi**
Add the following results to the top of the Lifestyle section
- Favourite programming language
- Favourite editor
- Favourite Mobile OS

### CODE REVIEW

File: data/lifestyle.ts [hunk 1]
```diff
@@ -217,6 +217,81 @@ let PARTIES = [{
   "value": 10,
 }];
 
+let PROGRAMMING_LANGUAGE = [{
+  "name": "Brainf*ck",
```
Comment (jaxonL) [inline data/lifestyle.ts:327]: wow xD was this troll

File: content/lifestyle.html [hunk 2]
```diff
@@ -1,5 +1,52 @@
...
+<h3> Tech Habits </h3>
+<p class="half">
+  <span class="big">
+    Python is the most popular programming language.
+  </span>
+  Python has diverse applications such as prototyping, web applications, machine learning, and scripting.
+  These are common areas faced by Software Engineers, making Python an ideal general purpose language that is easy to pick up.
+  <br/> <br/>
+  C++ is one of the first languages that SE students encounter in their courses.
+  It provides a strong programming foundation and is highly portable making it an ideal choide for multi-platform development.
```
Comment (jaxonL) [inline content/lifestyle.html]:
```suggestion
  It provides a strong programming foundation and is highly portable, making it an ideal choice for multi-platform development.
```

File: content/lifestyle.html [hunk 3]
```diff
@@ -1,5 +1,52 @@
...
+  It also has applications in Enterprise codebases, so students may encounter it over Co-op as well.
+  <br/> <br/>
+  Many students encounter web development either at co-ops or for side projects.
```
Comment (jaxonL) [inline content/lifestyle.html]:
```suggestion
  Many students encounter web development either during co-op or for side projects.
```

File: content/lifestyle.html [hunk 4]
```diff
@@ -1,5 +1,52 @@
...
+<div class="half second">
+  <span class="graph-title">Favourite programming language</span>
+  <div id="programming-language"></div>
+</div>
+
+<p class="half">
+  <span class="big">
+    Visual Studio Code was the most popular editor. Trailing farther behind were Vim and IntelliJ.
+  </span>
+  Visual Studio Code provides a large extension and theme ecosystem which make it easy to use and set up for various types of projects.
```
Comment (jaxonL) [inline content/lifestyle.html]:
```suggestion
  Visual Studio Code provides a large extension and theme ecosystem which makes it easy to use and set up for various types of projects.
```

File: content/lifestyle.html [hunk 5]
```diff
@@ -1,5 +1,52 @@
...
+  <a rel="noopener" href="https://insights.stackoverflow.com/survey/2019#technology-_-most-popular-development-environments" target="_blank" > According to this Stack Overflow survey</a> VSCode also rated as the most popular development environment.
```
Comment (jaxonL) [inline content/lifestyle.html]:
```suggestion
  <a rel="noopener" href="https://insights.stackoverflow.com/survey/2019#technology-_-most-popular-development-environments" target="_blank" > According to this Stack Overflow survey</a> VSCode was also rated as the most popular development environment.
```

File: content/lifestyle.html [hunk 6]
```diff
@@ -1,5 +1,52 @@
...
+  <div id="editor"></div>
+</div>
+
+<p class="half">
+  <span class="big">
+    iOS was the favourite mobile OS, used among 63% of students.
+  </span>
+  Interestingly, Android dominates the <a rel="noopener" href="https://gs.statcounter.com/os-market-share/mobile/worldwide" target="_blank" >Worldwide market share</a>, running on around 72% of devices compared to 26% for iOS.
+  <br/> <br/>
+  However, in North America we see 53% of devices running iOS and 45% using Android. A similar proportion to what is seen among SE students.
```
Comment (jaxonL) [inline content/lifestyle.html]:
```suggestion
  However, in North America, we see 53% of devices running iOS and 45% using Android, which is a similar proportion to what is seen among SE students.
```

**Review by jaxonL — COMMENTED** [review]
small comments >< i don't see an update bundle.min.js; i think that's why the deploy preview can't display the graphs :x

File: data/lifestyle.ts (hunk 1 above)
Comment (nsabahi) [reply data/lifestyle.ts:327]: I don't think so lol. Seems to be a thing: https://en.wikipedia.org/wiki/Brainfuck

**Comment by nsabahi** [comment]
Thanks, I forgot to upload the build files! Fixed those typos too

File: data/lifestyle.ts (hunk 1 above)
Comment (jaxonL) [reply data/lifestyle.ts:327]: ye i've heard of brainfk; i meant the people who'd actually put this in haha

**Review by jaxonL — APPROVED** [review]
sorry about that earlier comment on bundle.min 😢 guess we're removing it again 😓 

otherwise, LGTM 💻

**Pull request merged**